frontend/src/composables/errorService.ts

```diff
@@ -0,0 +1,36 @@
+import {reactive} from 'vue';
+import {AxiosError} from "axios";
+import {useToastService} from "@/composables/toastService";
+
+export function useErrorService() {
+
+  const errorState = reactive({
+    errorCode: '',
+    errorMessage: '',
+    // some other properties...
```

Comment:
This comment is mysterious :grin: Does it convey any useful information?

Reply:
As I said, the current implementation I've outlined should be treated more like a sandbox than a concrete thing :) I left this comment to get your attention and maybe get some ideas on what else we'd like to include here if anything ;)